engine/src/main/java/io/camunda/zeebe/engine/processing/bpmn/behavior/BpmnEventPublicationBehavior.java

```diff
@@ -113,20 +113,22 @@ public Optional<CatchEventTuple> findEscalationCatchEvent(
    * event if the given element instance is exists and is accepting events, e.g. isn't terminating,
    * wasn't interrupted, etc.
    *
-   * @param element the instance of the intermediate throw event
+   * @param throwElementId the element id of the escalation throw event
+   * @param escalation the escalation of throw event
    * @param activated process instance-related data of the element that is executed
    * @return returns true if the escalation throw event can be completed, false otherwise
    */
   public boolean throwEscalationEvent(
-      final ExecutableIntermediateThrowEvent element, final BpmnElementContext activated) {
-    final var escalation = element.getEscalation();
+      final DirectBuffer throwElementId,
+      final ExecutableEscalation escalation,
+      final BpmnElementContext activated) {
```

Comment:
🙃 This is somewhat nitpicky, but I would prefer renaming this from `activated` to `context`. This behavior shouldn't have to care at what stage this method is called. If we want to throw an escalation event during completion of an element we can use the same method, though the name doesn't make sense in that case.